cernopendata/modules/fixtures/data/records/cms-dimuon-spectrum-2011.json

```diff
@@ -50,24 +50,35 @@
+    "description": " <p>If you do not have the CERN Virtual Machine for 2011 CMS data installed, follow the instructions in step 1 at <a href=\"/VM/CMS/2011\">How to install a CERN Virtual Machine</a>.</p> <p>To run the example, you can follow the instructions and download all files needed for it from  <a href=\"https://github.com/cms-opendata-analyses/DimuonSpectrum2011\">https://github.com/cms-opendata-analyses/DimuonSpectrum2011</a>. </p> <p>You may prefer to setup the example with the detailed instruction below, if you want to get insight on how to build your own analysis and what are the files needed. In this case, install and run the Demo (demo analyzer) program as instructed in <a href=\"/docs/cms-virtual-machine-2011#test\">step 2 at 'How to install a CERN Virtual Machine'</a>. Then <ol> <li>Create directory <code>datasets</code> under <code>Demo/DemoAnalyzer</code></li>\n <li>Download the index files for the <a href=\"/record/17\">/DoubleMu/Run2011A-12Oct2013-v1/AOD</a> primary dataset (default). Only the first index file is actually needed for initial running. You can also use other primary datasets containing muons from 2011 or 2012.</li>\n <li>Replace the three files <code>BuildFile.xml</code>, <code>demoanalyzer_cfg.py</code>, <code>src/DemoAnalyzer.cc</code> with the ones from this record and read the comments in <code>DemoAnalyzer.cc</code> for more details.</li>\n <li>Recompile (<kbd>scram b</kbd>) and run (<kbd>cmsRun ...</kbd>).</ol></p>  <p>You should get an output file <kbd>DoubleMu.root</kbd>, which contains histograms for 10000 input events (a small subset of the data). These can be looked at using a ROOT Browser. The most interesting histogram is <kbd>GM_mass_log</kbd>. In order to compare it with the invariant-mass spectrum of di-muons in <a href=\"https://inspirehep.net/record/1118729/plots\">MUO-10-004</a>, it should be viewed with the <kbd>logy</kbd> option. With 10000 events the comparison is poor, but the J/&psi; (at log10(mass)=0.5), &Upsilon; (at log10(mass=0.98)) and Z (at log10(mass)=1.95, one event only) peaks should be visible. Furthermore, since the triggers for 2010 and 2011, as well as SingleMu and DoubleMu are different, the general shape of the distributions will differ somewhat depending on input.</p> <p>Finally, to run over more or even the full data, edit the relevant parts of the Python file <kbd>demoanalyzer_cfg.py</kbd> (see comments therein) and rerun.\n</p> "
   },
   "use_with": {
-    "description": "Use this with 2011 Mu primary datasets or any other 2011 primary datasets containing muons (see detailed instructions in the code)."
+    "description": "Use this with 2011 or 2012 primary datasets containing muons.",
+    "links": [
+      {
```

Comment:
You cannot have one link with several "recid" fields inside, you have to create separate "link" instance, for example like this:

```
    "links": [
      {
        "recid": "5101"
      },
      {
        "recid": "5102"
      },
      {
        "recid": "5100"
      }
    ]
```

see e.g. `cms-masterclass-files.json` for example.
```

Reply:
Whoops. Fixed.